src/View/Helper/DownloadsHelper.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Tatoeba Project, free collaborative creation of multilingual corpuses project
+ * Copyright (C) 2020 Tatoeba Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+namespace App\View\Helper;
+
+use App\View\Helper\AppHelper;
+use App\Model\Entity\LanguageNameTrait;
+
+class DownloadsHelper extends AppHelper
+{
+    use LanguageNameTrait;
+
+    public $helpers = ['Html', 'Languages'];
+
+    const PER_LANGUAGE_DIR = '/var/www-downloads/exports/per_language';
+    const DOWNLOAD_URL = 'https://downloads.tatoeba.org/exports';
```

Comment:
We have the habit of putting this kind of constants in `app_local.php`, so that it’s easier to integrate with Imouto or any other custom instance. If you go for this, please add new variables in `app_local.php.template` that can be readily used by Imouto. Within your tests, you can then add the appropriate `Configure::write()` calls in the `setUp()` method.